lib/improver/utilities/cube_manipulation.py

```diff
@@ -228,7 +230,7 @@ def merge_cubes(cubes):
     if isinstance(cubes, iris.cube.Cube):
         cubes = iris.cube.CubeList([cubes])
 
-    cubelist = equalise_cubes(cubes)
+    cubelist = equalise_cubes(cubes, model_id_attr, merging=True)
```

Comment:
We shouldn't rely on position for a keyword argument.
```suggestion
    cubelist = equalise_cubes(cubes, model_id_attr=model_id_attr, merging=True)
```

Reply:
Done.